tools/fq.wdl

```diff
@@ -53,3 +53,68 @@ task fqlint {
         read2: "Input FastQ with read two"
     }
 }
+
+task subsample {
+    meta {
+        description: "This WDL task subsamples input FastQs"
+        outputs: {
+            subsampled_read1: "Subsampled read1 FastQ"
+            subsampled_read2: "Subsampled read2 FastQ"
+        }
+    }
+
+    parameter_meta {
+        read1: "Input FastQ with read one"
+        read2: "Input FastQ with read two"
+        prefix: "Prefix for the output FastQ file(s). The extension `_R1.subsampled.fastq.gz` and `_R2.subsampled.fastq.gz` will be added."
+        probability: "The probability a record is kept, as a percentage (0.0, 1.0). Cannot be used with `record-count`. Any `probability<=0.0` or `probability>=1.0` to disable."
+        record_count: "The exact number of records to keep. Cannot be used with `probability`. Any `record_count<=0` to disable."
```

Comment:
Is this enforced in our task?

Reply:
No, I'm relying on `fq` to produce an informative error if both are supplied:
```
$ fq subsample -p 0.5 -n 100
error: the argument '--probability <PROBABILITY>' cannot be used with '--record-count <RECORD_COUNT>'
```